enterprise/cli/root.go

```diff
@@ -1,28 +1,37 @@
 package cli
 
 import (
-	"github.com/coder/coder/v2/cli"
+	agplcli "github.com/coder/coder/v2/cli"
 	"github.com/coder/serpent"
 )
 
 type RootCmd struct {
-	cli.RootCmd
+	agplcli.RootCmd
 }
 
 func (r *RootCmd) enterpriseOnly() []*serpent.Command {
 	return []*serpent.Command{
+		// Duplicates from AGPL:
```

Comment:
Let's rather reference a canonical list instead of relying on keeping this list in sync.

Reply:
This is a misunderstanding, these are commands that have different implementations to AGPL. Will improve the comment